mopidy/models/serialize.py

```diff
@@ -2,7 +2,9 @@
 
 import json
 
-from mopidy.models.immutable import ImmutableObject
+from mopidy.models import immutable
+
+_MODELS = ['Ref', 'Artist', 'Album', 'Track', 'TlTrack', 'Playlist']
```

Comment:
Should be `_DESERIALIZABLE_MODELS` to be honest. Currently we have no API that should need this, though I can't be that say mopidy-podcast doesn't stash things on disk using us. So if this is a case we will need to revert this and look expanding all sub-classes of `ImmutableObject`.

Other option could be having a registry approach, for instance using ABC or just have a factory for creating a decoder instance that knows about the extra models you pass it.

But for now I would say postpone or revert to using the sub-classes. Also note that the sub-class case would break as both Mopdidy and Mopidy-Podcast have different `Image` models.